plugins.txt

```diff
@@ -1,21 +1,22 @@
 docker-workflow:1.9.1
 ansible:0.6.2
-pipeline-milestone-step:1.2
-envinject:2.1.3
-aws-credentials:1.16
-credentials-binding:1.10
-ws-cleanup:0.32
-jobConfigHistory:2.15
+envinject:2.1.4
+aws-credentials:1.23
+credentials-binding:1.13
+ws-cleanup:0.34
+jobConfigHistory:2.18
 copyartifact:1.38.1
-s3:0.10.10
-ssh-agent:1.13
-extended-choice-parameter:0.75
+s3:0.10.12
+bouncycastle-api:2.16.2
+ssh-agent:1.15
+extended-choice-parameter:0.76
 rebuild:1.25
-ansicolor:0.5.0
-permissive-script-security:0.1
+ansicolor:0.5.2
+permissive-script-security:0.3
 swarm:3.4
-github-branch-source:2.0.5
-config-file-provider:2.15.7
-cobertura:1.10
-http_request:1.8.20
-ssh-slaves:1.21
\ No newline at end of file
+github-branch-source:2.2.3
+config-file-provider:2.16.4
+cobertura::1.11
+http_request:1.8.21
+ssh-slaves:1.21
+maven-plugin:2.17
```

Comment:
I think I'll remove this actually. I believe I installed it as an attempt to get rid of some stack traces during startup.